### ISSUE TITLE
[BUG]Max belt Problem

### ISSUE THREAD

**Issue by MohamedElgharieb** [asim97/SecureCodingDojo_DEBI#19]
**Describe the bug**
-The Value of max belt is added 1 to the instructor UI after updating the value
-the following line has a syntax error:
```javascript
    studentUpdates.solution_disabled!='disabled';
``` 
it should be:
```javascript
    studentUpdates.solution_disabled='disabled';
```
file : trainingportal/server.js
line:624

**To Reproduce**
1. Log in as instructor account
2. Change the max belt (Screen Shoot 1)
3. press the update button
4. after refreshing the page you will see the max belt value increased by 1 (Screen Shoot 2)

**Expected behavior**
The following lines:
``` html
<select class="form-select"  name="challengelevel_limit" id="challengelevel_limit"> 
    <option value=1 ng-selected="studentEntry.max_progress==0" >Yellow Belt</option>
    <option value=2 ng-selected="studentEntry.max_progress==1" >Orange Belt</option>
```
Should be:
``` html
<select class="form-select"  name="challengelevel_limit" id="challengelevel_limit"> 
    <option value=1 ng-selected="studentEntry.max_progress==1" >Yellow Belt</option>
    <option value=2 ng-selected="studentEntry.max_progress==2" >Orange Belt</option>
```

**Screenshots**
(Screen Shoot 1)
![image](https://user-images.githubusercontent.com/13475515/124047249-07344800-da14-11eb-8c29-1c745e01ce9d.png)
(Screen Shoot 2)
![image](https://user-images.githubusercontent.com/13475515/124132167-d1797880-da80-11eb-8c50-6dcb01a6e0d4.png)

**Comment by asim97** [comment]
Thanks for your investigation.... this will be fixed asap

**Comment by asim97** [comment]
Fixed in the comit c78a809b085d4d04c0e9eb862295afb970a94528
